### PR TITLE
Azure tutorials

### PR DESCRIPTION
**Pull request by rushah**
A few tutorials to get started working with Microsoft Azure and StreamSets Data Collector.

### CODE REVIEW

File: working-with-azure/blobstorage_to_hdinsightkafka.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+When working with HDInsight Kafka, we need to pre-create the Kafka topic. Ssh into the terminal for StreamSets Data Collector on HDInsight and run the following command to create a Kafka topic. Remember to replace the zookeeper_url with your own uri for zookeeper.
+
+	/usr/hdp/current/kafka-broker/bin/kafka-topics.sh --create --replication-factor 3 --partitions 3 --topic nyctaxi --zookeeper <zookeeper_uri>
+
+## Creating a Pipeline
+
+Now let's get some data flowing! In your browser, login to StreamSets Data Collector (SDC) and create a new pipeline.
+
+Select Origin from the Drop down list: `Hadoop FS Standalone`
```
Comment (metadaddy) [inline working-with-azure/blobstorage_to_hdinsightkafka.md]: Probably need to explicitly mention HDP and the version here, even though you show it in the screenshot.

File: working-with-azure/hdinsightkafka_to_sqldw_and_blobstorage.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
+# Ingesting Data from HDInsight Kafka into Azure SQL Datawarehouse and Hive backed by Blob Storage
```
Comment (metadaddy) [inline working-with-azure/hdinsightkafka_to_sqldw_and_blobstorage.md]:
```suggestion
# Ingesting Data from Apache Kafka on HDInsight into Azure SQL Data Warehouse and Apache Hive backed by Azure Blob Storage
```

File: working-with-azure/blobstorage_to_hdinsightkafka.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+If all is well, it’s time to run the pipeline! Hit the run button ![image alt text](img/BlobToKafka/PlayIcon.png) and you should see 5386 input records and 5386 output records in the monitoring panel.
+
+![image alt text](img/BlobToKafka/StartandMonitor.png)
+
+Use Kafka tools provided by Azure to verify data was written to the topic. More details can be found here: [Manage Kafka Topics](https://docs.microsoft.com/en-us/azure/hdinsight/kafka/apache-kafka-get-started#manage-kafka-topics)
+
+## Conclusion
+
+This tutorial shows how simple it is to stream data from any Directory on Blob Storage, structured on unstructured, into a Kafka topic that can feed multiple consumers downstream. 
+Follow the next tutorial to see how you can use StreamSets Data Collector to read data from the topic and feed it into a SQL Datawarehouse on Azure.
```
Comment (metadaddy) [inline working-with-azure/blobstorage_to_hdinsightkafka.md]:
```suggestion
Follow the next tutorial to see how you can use StreamSets Data Collector to read data from the topic and feed it into Azure SQL Data Warehouse.
```

File: working-with-azure/hdinsightkafka_to_sqldw_and_blobstorage.md [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+
+[Kafka](https://docs.microsoft.com/en-us/azure/hdinsight/kafka/apache-kafka-introduction) is an open-source, distributed streaming platform. It's often used as a message broker, as it provides functionality similar to a publish-subscribe message queue. [HDInsight Kafka](https://azure.microsoft.com/en-us/services/hdinsight/apache-kafka/) is a managed service that provides a simplified configuration for Apache Kafka.
```
Comment (metadaddy) [inline working-with-azure/hdinsightkafka_to_sqldw_and_blobstorage.md]:
```suggestion
[Kafka](https://docs.microsoft.com/en-us/azure/hdinsight/kafka/apache-kafka-introduction) is an open-source, distributed streaming platform. It's often used as a message broker, as it provides functionality similar to a publish-subscribe message queue. [Apache Kafka on HDInsight](https://azure.microsoft.com/en-us/services/hdinsight/apache-kafka/) is a managed service that provides a simplified configuration for Apache Kafka.
```

File: working-with-azure/blobstorage_to_hdinsightkafka.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
+# Ingesting Data from Blob Storage into HDInsight Kafka
```
Comment (metadaddy) [inline working-with-azure/blobstorage_to_hdinsightkafka.md]:
```suggestion
# Ingesting Data from Blob Storage into Apache Kafka on HDInsight
```

File: working-with-azure/blobstorage_to_hdinsightkafka.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+
+[Azure Blob Storage](https://azure.microsoft.com/en-us/services/storage/blobs/ "Azure Blob Storage") can handle all your unstructured data, scaling up or down as required. You no longer have to manage it, you only pay for what you use, and you save money over on-premises storage options.
+
+[Kafka](https://docs.microsoft.com/en-us/azure/hdinsight/kafka/apache-kafka-introduction) is an open-source, distributed streaming platform. It's often used as a message broker, as it provides functionality similar to a publish-subscribe message queue. [HDInsight Kafka](https://azure.microsoft.com/en-us/services/hdinsight/apache-kafka/) is a managed service that provides a simplified configuration for Apache Kafka.
```
Comment (metadaddy) [inline working-with-azure/blobstorage_to_hdinsightkafka.md]:
```suggestion
[Apache Kafka](https://docs.microsoft.com/en-us/azure/hdinsight/kafka/apache-kafka-introduction) is an open-source, distributed streaming platform. It's often used as a message broker, as it provides functionality similar to a publish-subscribe message queue. [Apache Kafka on HDInsight](https://azure.microsoft.com/en-us/services/hdinsight/apache-kafka/) is a managed service that provides a simplified configuration for Apache Kafka.
```

File: working-with-azure/blobstorage_to_hdinsightkafka.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+
+## Goal
+
+In this tutorial, you will learn how to leverage Azure services with StreamSets to read data from Blob Storage and send data into HDInsight Kafka cluster.
```
Comment (metadaddy) [inline working-with-azure/blobstorage_to_hdinsightkafka.md]:
```suggestion
In this tutorial, you will learn how to leverage Azure services with StreamSets to read data from Blob Storage and send data into Apache Kafka on HDInsight cluster.
```

File: working-with-azure/hdinsightkafka_to_sqldw_and_blobstorage.md [hunk 8]
```diff
@@ -0,0 +1,187 @@
...
+
+[SQL Data Warehouse](https://docs.microsoft.com/en-us/azure/sql-data-warehouse/sql-data-warehouse-overview-what-is) is a cloud-based Enterprise Data Warehouse (EDW) that leverages Massively Parallel Processing (MPP) to quickly run complex queries across petabytes of data. Use SQL Data Warehouse as a key component of a big data solution. SQL Data Warehouse stores data into relational tables with columnar storage. Once data is stored in SQL Data Warehouse, you can run analytics at massive scale.
```
Comment (metadaddy) [inline working-with-azure/hdinsightkafka_to_sqldw_and_blobstorage.md]:
```suggestion
[Azure SQL Data Warehouse](https://docs.microsoft.com/en-us/azure/sql-data-warehouse/sql-data-warehouse-overview-what-is) is a cloud-based Enterprise Data Warehouse (EDW) that leverages Massively Parallel Processing (MPP) to quickly run complex queries across petabytes of data. Use SQL Data Warehouse as a key component of a big data solution. SQL Data Warehouse stores data into relational tables with columnar storage. Once data is stored in SQL Data Warehouse, you can run analytics at massive scale.
```

File: working-with-azure/hdinsightkafka_to_sqldw_and_blobstorage.md [hunk 9]
```diff
@@ -0,0 +1,187 @@
...
+
+[Azure Blob Storage](https://azure.microsoft.com/en-us/services/storage/blobs/ "Azure Blob Storage") can handle all your unstructured data, scaling up or down as required. You no longer have to manage it, you only pay for what you use, and you save money over on-premises storage options.
+
+## Goal
+
+In this tutorial, you will learn how to leverage Azure services with StreamSets to read data from an HDInsight Kafka stream into ingest into Azure SQL Datawarehouse and a Hive table backed by Blob Storage.
```
Comment (metadaddy) [inline working-with-azure/hdinsightkafka_to_sqldw_and_blobstorage.md]:
```suggestion
In this tutorial, you will learn how to leverage Azure services with StreamSets to read data from an Apache Kafka on HDInsight stream into ingest into Azure SQL Datawarehouse and a Hive table backed by Blob Storage.
```

File: working-with-azure/hdinsightkafka_to_sqldw_and_blobstorage.md [hunk 10]
```diff
@@ -0,0 +1,187 @@
...
+
+## Prerequisites
+
+## Creat the SQL Datawarehouse table
```
Comment (metadaddy) [inline working-with-azure/hdinsightkafka_to_sqldw_and_blobstorage.md]:
```suggestion
## Creat the SQL Data Warehouse table
```

File: working-with-azure/readme.md [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+
+2. Create a [Virtual Network](https://docs.microsoft.com/en-us/azure/virtual-network/quick-create-portal#create-a-virtual-network) (vNet):
+This vNet will allow us to enable communication between the clusters created in the next steps to communicate privately with each other.
+
+3. Data storage: Create a [Blob Container](https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-hadoop-use-blob-storage#create-blob-containers)
+
+4. Install [StreamSets Data Collector for HDInsight](https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-apps-install-streamsets)
+Also check [StreamSets Documentation](https://streamsets.com/documentation/datacollector/latest/help/index.html#datacollector/UserGuide/Installation/CloudInstall.html#task_vnj_rl2_wdb) for more details on the installation process.
+
+5. Create an [HDInsight Kafka cluster](https://docs.microsoft.com/en-us/azure/hdinsight/kafka/apache-kafka-get-started) in the same vNet as above. Use Azure Storage for the Kafka cluster.
```
Comment (metadaddy) [inline working-with-azure/readme.md]:
```suggestion
5. Create an [Apache Kafka on HDInsight cluster](https://docs.microsoft.com/en-us/azure/hdinsight/kafka/apache-kafka-get-started) in the same vNet as above. Use Azure Storage for the Kafka cluster. 
```

File: working-with-azure/hdinsightkafka_to_sqldw_and_blobstorage.md [hunk 12]
```diff
@@ -0,0 +1,187 @@
...
+
+
+## Running the Pipeline
+
+If all is well, it’s time to run the pipeline! Hit the run button ![image alt text](img/BlobToKafka/PlayIcon.png) and you should see 5386 input records and 5386 output records in the monitoring panel.
+
+
+## Conclusion
+
+This tutorial shows how simple it is to stream data from any Kafka into multiple destinations with Azure Services. For Realtime trickle feeds, you can directly send data into SQL Datawarehouse and for Archival, you can send data into Hive.
```
Comment (metadaddy) [inline working-with-azure/hdinsightkafka_to_sqldw_and_blobstorage.md]:
```suggestion
This tutorial shows how simple it is to stream data from any Kafka topic into multiple destinations with Azure Services. For real-time trickle feeds, you can directly send data into Azure SQL Data Warehouse and for archival, you can send data into Hive.
```

File: working-with-azure/blobstorage_to_hdinsightkafka.md (hunk 1 above)
Comment (rushah) [reply working-with-azure/blobstorage_to_hdinsightkafka.md]: Good point.

**Pull request merged**